user-guide/mkdocs.yml

```diff
@@ -110,9 +110,9 @@ nav:
 
   - Use Cases:
     - Overview: usecases/overview.md
-    - Data Analytics:
-      - Basic Image Browsing and Mapping: usecases/haan/usecase.md
-      - Grouping and Tagging Image Files: usecases/haan/usecase-2.md
+# the following have been moved to the user guide. we need to remove these files from the directory in use cases
```

Comment:
```suggestion
    - Data Analytics:
# the following have been moved to the user guide. we need to remove these files from the directory in use cases
```

<details><summary>I believe the "Data Analytics" header is still expected.</summary>

The `mkdocs serve` command fails without it, because `- Overview` points to a string, then an array appears. (A nav property must point to either a string or an array.)

</details>

I restore it.